### DISCUSSION TITLE
Is something wrong with my laptop?

### DISCUSSION THREAD

**Discussion by mrfirst** [intel/AI-Playground#49] [Q&A] [answered]
Help! the software can't work.
<img width="640" alt="屏幕截图 2024-09-08 024052" src="https://github.com/user-attachments/assets/da67fc21-c89c-4ba1-9789-4f0e16d6499e">
<img width="428" alt="屏幕截图 2024-09-08 024137" src="https://github.com/user-attachments/assets/454dd231-fd7c-4f5a-bd5d-7b8dd9117f9d">

**Comment by Nuullll** [comment]
What's your GPU driver version?
And make sure you are using the installer for MTL: https://github.com/intel/AI-Playground/releases/download/v1.01beta.mtl/AI.Playground-v1.01b-Mobile_MTL-H.exe

**Reply by mrfirst** [reply]
I'm sure that I have installed the MTL version. Attachment is my GPU driver vision.
<img width="392" alt="屏幕截图 2024-09-10 115543" src="https://github.com/user-attachments/assets/8779fa80-93e6-4605-a305-7414fb85067f">

**Reply by mrfirst** [reply, marked as answer]
Hi, I have solved it by upgraded My pc memory from 16GB to 32GB. Thanks!

**Comment by lh3953** [comment]
在打开程序的时候 一直提示AI playground loading，怎么解决？

**Reply by mrfirst** [reply]
If you were behand the wallway's great firewall, you would need some technology and pay any attention.

**Comment by lh3953** [comment]
你说的防火墙是指中国对外访问的网络防火墙还是指我自己电脑的防火墙？ 如果是中国防火墙，我使用vpn还是无法使用。如果是我电脑上的，我防火墙处于关闭状态。

**Comment by ccc0ra** [comment]
### Quality Regression in SDXL Image Generation After Latest Intel® Graphics Driver & NPU Drivers

The latest driver updates for my Intel system (Ultra 7 255H, Arc 140V NPU – _Lenovo Intel Aura Edition_) have caused a dramatic regression in SDXL image generation quality. I’m not referring to resolution or definition — those remain fine — but to the core aspects of generation: relevance, creativity, coherence, stability of compositions, or prompt interpretation.

Before updating, SDXL on my Intel machine (comfy via AIPG) consistently produced results that were significantly better than what I can achieve today, even when testing on a high‑end NVIDIA L40S instance. The previous Intel AI Playground pipeline was outperforming NVIDIA hardware in terms of prompt fidelity, artistic creativity, and structural stability.

After the update, SDXL outputs on my Intel system became similar to the one of my NVIDIA L40S Pod.
I attempted a full Windows restore and reinstalled older drivers, but the issue persists. It seems the update installed or modified additional components (possibly third‑party or internal dependencies) that cannot be rolled back. I’m now stuck with a system that no longer performs anywhere near what it used to.

For reference, the two drivers I updated simultaneously were:
**Intel® Graphics Driver 32.0.101.8974 (WHQL Certified)
Intel® NPU Driver – Windows 32.0.100.4841***

I don’t know which one introduced the regression, but the change is severe enough to make me cry.

Unfortunately, I was banned from the Intel Discord server without explanation. I never insulted anyone or behaved aggressively — I only asked technical questions about SDXL training limitations. I have no other way to reach the responsible teams, so I’m hoping this message can be forwarded to the engineers working on the latest driver stack.

The previous Intel AI Playground (I was on the v2.x but since my update, I tried with the v2 and the v3 but it doesn't give me back my old performances.

Thank you for your time, and I hope this can be investigated.
*if anyone can just copy/paste my message in the dicord channel...
